### PR TITLE
fixing issues identified in Issue 48

### PR DESCRIPTION
**Pull request by sjhitchner**

### CODE REVIEW

File: resolver_plugin.go [hunk 1]
```diff
@@ -86,11 +86,17 @@ func (m *ResolverPlugin) generateSingleFile(data *codegen.Data, models []*Model,
 		Alias:      "dm",
 		ImportPath: path.Join(m.rootImportPath, m.backend.Directory),
 	})
+
 	file.Imports = append(file.Imports, Import{
 		Alias:      "fm",
 		ImportPath: path.Join(m.rootImportPath, m.frontend.Directory),
 	})
 
+	file.Imports = append(file.Imports, Import{
```
Comment (RichardLindhout) [inline resolver_plugin.go:95]: This is not needed in my opinion as this is the same as the "fm" directory?

Comment (sjhitchner) [reply resolver_plugin.go:95]: The code will not compile without importing the `Resolvers` from generated graphql code.  They are not the same directory in my case.  The fm package contains the graphql model code.  The gm package contains the generated graphql code.

These need to be imported from the generated graphql code.
```
func (r *Resolver) Mutation() gm.MutationResolver	{ return &mutationResolver{r} }
func (r *Resolver) Query() gm.QueryResolver		{ return &queryResolver{r} }

type mutationResolver struct{ *Resolver }
type queryResolver struct{ *Resolver }
```

Perhaps there is an issue with directory configuration/assumptions.

**Comment by RichardLindhout** [comment]
Ah I didn't know , you're totally right!

**Comment by RichardLindhout** [comment]
Thanks a lot for your changes will release a prerelease and try to make it official as soon as I work on it again :)

**Pull request merged**